src/services/shortLink/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Context, GraaspPlatform } from '@graasp/sdk';
+
+import { CLIENT_HOSTS, CLIENT_PREFIX } from '../../utils/config';
+
+export function getPlatformHost(shortLinkView: GraaspPlatform) {
+  const [BUILDER_HOST, PLAYER_HOST, LIBRARY_HOST] = CLIENT_HOSTS;
+
+  switch (shortLinkView) {
+    case GraaspPlatform.BUILDER:
+      return BUILDER_HOST;
+    case GraaspPlatform.LIBRARY:
+      return LIBRARY_HOST;
+    case GraaspPlatform.PLAYER:
+      return PLAYER_HOST;
+    default:
+      throw new Error('Invalid Shortlink view');
+  }
+}
+
+export function getUrlHost({ url }: { name: Context; url: URL }) {
+  const urlStr = url.toString();
+  // if the url ends with /, remove it
+  if (urlStr.endsWith('/')) {
+    return urlStr.slice(0, -1);
+  }
+  return urlStr;
+}
+
+export function getPlatformPrefix(shortLinkView: GraaspPlatform) {
+  // TODO: set as const
+  const [BUILDER_PREFIX, PLAYER_PREFIX, LIBRARY_PREFIX] = CLIENT_PREFIX;
+
+  switch (shortLinkView) {
+    case GraaspPlatform.BUILDER:
+      return BUILDER_PREFIX;
+    case GraaspPlatform.LIBRARY:
+      return LIBRARY_PREFIX;
+    case GraaspPlatform.PLAYER:
+      return PLAYER_PREFIX;
+    default:
+      throw new Error('Invalid Shortlink view');
+  }
+}
```

Comment:
Do we have something like this in sdk ? If not, maybe this would be something interesting as a source of truth for both front and back.